src/events/message/index.ts

```diff
@@ -24,45 +27,82 @@ export default class MessageEvent extends Event {
 		}
 
 		if (message.attachments.size > 0) {
```

Comment:
```suggestion
```suggestion

		const attachment = message.attachments.find((attachment) => {
			if (!attachment.name || attachment.width) {
				return false;
			}
			const fileExtension = extname(attachment.name).substring(1);
			const language = fileExtension || "txt";

			return language === "txt" || extensions.has(language);
		})

		if (attachment) {
```